hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -1354,9 +1354,9 @@ message UpdateGetS3SecretRequest {
   This will be used by OM to authenticate S3 gateway requests on a per request basis.
 */
 message S3Authentication {
-    optional string stringToSign = 1;
-    optional string signature = 2;
-    optional string accessId = 3;
+    required string stringToSign = 1;
+    required string signature = 2;
+    required string accessId = 3;
```

Comment:
Should keep these `optional`? @kerneltime

Reply:
We should keep these as optional. Any error handling should be at the higher layer where the request is processed. Here the validation will fail if set incorrectly in a secure cluster.

Reply:
I believe the best approach is to keep the protobuf layer simple and enforce at the higher layer.